TrainingExtensions/tensorflow/src/python/aimet_tensorflow/keras/utils/common.py

```diff
@@ -611,6 +611,13 @@ def get_number_of_outputs_and_axis_handling(layer, weight_shape, param_type) ->
     :param param_type: str
     :return: Tuple[int, int]
     """
+    # Raising an assertion error incase there's SeparableConv2D because in this case we have two sets of weights: Depthwise
```

Comment:
This utility function is not the right place to raise exceptions IMO. Policy decision - e.g. whether to raise exceptions, or handle gracefully or do something else should be taken by higher level logic - not by utility function code.

Reply:
Can we please move this exception raising code to QuantizationSimModel or some such higher level entity? Essentially what is the purpose of raising the exception - we are raising it because we don't handle models with separable layers - so its an unrecoverable error?

Reply:
Hi @quic-akhobare 
I moved the exception raising logic to `validate_model()` which is being called inside QuantizationSimModel. Can you please have a look.